### DISCUSSION TITLE
How to trigger and add few extra custom notification to portal bell Icon other than current default notification?

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Lern/Community#201] [Documentation]
Hello @AmiableAnil, @rhwarrier, @BharathwajShankar, 

I'm creating a new SunbirdEd react app where I want to achieve the following things:
`
1. What API and services are used to get the feeds/notifications in the app?
2. If we want to generate a customised notification for different events along with current notifications, then how and what will be the steps to achieve it?
3. Can we trigger notifications just with API? or any code customisation is needed?

cc: @paritshivani

**Comment by nidhikachhawa24** [comment]
Hello @pallakartheekreddy, @HarishGangula, @swayangjit @vinukumar-vs @rajeevsathish

Can we get guidance here Please??

cc: @komalm

**Reply by paritshivani** [reply]
Hello falks!

Any update here?? Waiting for some guidance here.

@pallakartheekreddy, @HarishGangula, @swayangjit @vinukumar-vs @rajeevsathish
